Compute/containers/recipes/kubernetes/terraform/main.tf

```
@@ -36,20 +36,46 @@ locals {
     local.dapr_config_name != null ? { "dapr.io/config" = local.dapr_config_name } : {}
   ) : {}
 
-  # Connections - used for linked resources like persistent volumes
+  # Connections - merged resource data from linked resources
   connections = try(var.context.resource.connections, {})
 
-  # Connection-derived environment variables, enabled when disableDefaultEnvVars is false
+  # Connection definitions - original connection config (has source, disableDefaultEnvVars)
+  connection_definitions = try(var.context.resource.properties.connections, {})
+
+  # Properties to exclude from connection environment variables
+  excluded_properties = ["application", "environment", "recipe", "status", "provisioningState"]
```

```suggestion
  excluded_properties = ["recipe", "status", "provisioningState"]
```
Applications.Core/containers creates environment variables for application and environment.